### PR TITLE
Update Login with WorkOS docs to use latest dashboard UI and best practices

### PR DESCRIPTION
**Pull request by zackproser-workos**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

Updates the docs and screenshots for Login with WorkOS to the latest best practices, including using [AuthKit](https://authkit.com).

## What is the current behavior?

The current docs reference out-of-date:
-  WorkOS dashboard screenshots 
- navigation instructions 
-  older versions of the WorkOS login experience

## What is the new behavior?

These changes update all WorkOS dashboard screenshots to reference the latest UI. It updates the documentation on Login with WorkOS to reference best practices, including the use of AuthKit for streamlined social, email-based, and single sign-on implementations that work out of the box.

## How I verified these changes

I started with [an official Supbase example application](https://github.com/supabase/supabase/tree/master/examples/auth/nextjs#clone-and-run-locally) and then I ran the supabase docs project locally from my branch containing these changes. 

I sequentially applied the modifications specified by the instructions updated in this pull request.

I verified that the following workflows succeeded end-to-end: 
- login
- user session persistence
- logout

## Additional context

### CODE REVIEW

File: apps/docs/content/guides/auth/social-login/auth-workos.mdx [hunk 1]
```diff
@@ -1,54 +1,53 @@
...
-description: 'Add WorkOS OAuth to your Supabase project'
+title: 'SSO and Social Login with WorkOS'
+description: 'Add WorkOS AuthKit to your Supabase project'
 ---
 
-To enable WorkOS Auth for your project, you need to set up WorkOS OAuth application and add the application credentials to your Supabase Dashboard.
+[WorkOS AuthKit](https://authkit.com) provides email/password authentication and social login by default. Once AuthKit is installed, you can enable SSO through a dashboard toggle.
 
-## Overview
+## Step 1. Create a WorkOS organization
```
Comment (PaulAsjes) [inline apps/docs/content/guides/auth/social-login/auth-workos.mdx]: Should step 1 (or step 0) be "Create a WorkOS account"?

File: apps/docs/content/guides/auth/social-login/auth-workos.mdx [hunk 2]
```diff
@@ -1,54 +1,53 @@
...
+## Step 2. Obtain your `Client ID` and `WORKOS_API_KEY` values
 
-Head over to the Configuration tab and configure the redirect URI.The redirect URI should look like `https://<project-ref>.supabase.co/auth/v1/callback`
-Note that this is distinct from the redirect URI referred to in the Supabase dashboard
+![Get your Environment's Client ID and Secret](/docs/img/guides/auth-workos/workos-dashboard-get-client-id-and-key.png)
 
-![Fetch Client ID and configure Redirect URI](/docs/img/guides/auth-workos/workos-clientid-redirect-uri.png)
+Visit the getting started page of the WorkOS dashboard, at [https://dashboard.workos.com/get-started](https://dashboard.workos.com/get-started), to obtain the following values from the Quickstart panel:
+* `WORKOS_CLIENT_ID`
+* `WORKOS_API_KEY`A
```
Comment (PaulAsjes) [inline apps/docs/content/guides/auth/social-login/auth-workos.mdx]:
```suggestion
* `WORKOS_API_KEY`
```

Comment (zackproser-workos) [reply apps/docs/content/guides/auth/social-login/auth-workos.mdx]: Thanks, fixed!

Comment (zackproser-workos) [reply apps/docs/content/guides/auth/social-login/auth-workos.mdx]: Thank you - I found and fixed this today, too.

File: apps/docs/content/guides/auth/social-login/auth-workos.mdx [hunk 3]
```diff
@@ -1,172 +1,129 @@
...
+[WorkOS AuthKit](https://authkit.com) is the world's best login box. AuthKit provides email/password authentication and social login by default. Once AuthKit is installed, you can configure and deploy additional SSO providers via the WorkOS dashboard.
 
-Log in to the dashboard and hop over to the Organizations tab to create and organization
-![Create an Organization](/docs/img/guides/auth-workos/workos-create-organization.png)
+## Option 1: Using WorkOS as a provider via Supabase auth 
+
+### Step 1. Create a WorkOS organization
 
-## Obtain the client ID and configure redirect URI
+Log in to the WorkOS dashboard and visit the Organizations tab to create an organization
```
Comment (PaulAsjes) [inline apps/docs/content/guides/auth/social-login/auth-workos.mdx:23]: Should we mention that you can do this programmatically via the API?

File: apps/docs/content/guides/auth/social-login/auth-workos.mdx [hunk 4]
```diff
@@ -1,172 +1,129 @@
...
+![Create an Organization](/docs/img/guides/auth-workos/workos-create-organization.png)
 
-Head over to the Configuration tab and configure the redirect URI.The redirect URI should look like `https://<project-ref>.supabase.co/auth/v1/callback`
-Note that this is distinct from the redirect URI referred to in the Supabase dashboard
+## Step 2. Obtain your `Client ID` and `WORKOS_API_KEY` values
```
Comment (PaulAsjes) [inline apps/docs/content/guides/auth/social-login/auth-workos.mdx:28]:
```suggestion
## Step 2. Obtain your `WORKOS_CLIENT_ID` and `WORKOS_API_KEY` values
```

File: apps/docs/content/guides/auth/social-login/auth-workos.mdx [hunk 5]
```diff
@@ -1,172 +1,129 @@
...
 
-![Fetch Client ID and configure Redirect URI](/docs/img/guides/auth-workos/workos-clientid-redirect-uri.png)
+![Get your Environment's Client ID and Secret](/docs/img/guides/auth-workos/workos-dashboard-get-client-id-and-key.png)
 
-## Obtain the WorkOS secret
+Visit the getting started page of the WorkOS dashboard, at [https://dashboard.workos.com/get-started](https://dashboard.workos.com/get-started), to obtain the following values from the Quickstart panel:
```
Comment (PaulAsjes) [inline apps/docs/content/guides/auth/social-login/auth-workos.mdx:32]: Should mention that you need to be signed in to see these.

File: apps/docs/content/guides/auth/social-login/auth-workos.mdx [hunk 6]
```diff
@@ -1,172 +1,129 @@
...
-```kotlin
-suspend fun signOut() {
-	supabase.auth.signOut()
-}
-```
+### Step 1. Install and configure AuthKit
 
-</TabPanel>
-</Tabs>
+Visit [https://authkit.com](https://authkit.com) to get up and running with AuthKit, the world's best login box.
```
Comment (PaulAsjes) [inline apps/docs/content/guides/auth/social-login/auth-workos.mdx]: I think we should link to the Quick Start directly rather than the marketing page.

**Review by PaulAsjes — CHANGES_REQUESTED** [review]

File: apps/docs/content/guides/auth/social-login/auth-workos.mdx (hunk 3 above)
Comment (zackproser-workos) [reply apps/docs/content/guides/auth/social-login/auth-workos.mdx:23]: Yeah good call - I'll add.

File: apps/docs/content/guides/auth/social-login/auth-workos.mdx (hunk 6 above)
Comment (zackproser-workos) [reply apps/docs/content/guides/auth/social-login/auth-workos.mdx]: Good call

Comment (zackproser-workos) [reply apps/docs/content/guides/auth/social-login/auth-workos.mdx]: Done in [54d86d0](https://github.com/zackproser-workos/supabase/pull/1/commits/54d86d0d34e45c7f55496e723c2fa235fc0cf083)

File: apps/docs/content/guides/auth/social-login/auth-workos.mdx (hunk 5 above)
Comment (zackproser-workos) [reply apps/docs/content/guides/auth/social-login/auth-workos.mdx:32]: Done in [54d86d0](https://github.com/zackproser-workos/supabase/pull/1/commits/54d86d0d34e45c7f55496e723c2fa235fc0cf083)

File: apps/docs/content/guides/auth/social-login/auth-workos.mdx (hunk 3 above)
Comment (zackproser-workos) [reply apps/docs/content/guides/auth/social-login/auth-workos.mdx:23]: Added in [54d86d0](https://github.com/zackproser-workos/supabase/pull/1/commits/54d86d0d34e45c7f55496e723c2fa235fc0cf083)

**Comment by zackproser-workos** [comment]
Closing in favor of https://github.com/supabase/supabase/pull/30451

**Pull request closed without merging**